Hardware/Computer.cs

```diff
@@ -162,10 +171,10 @@ public class Computer : IComputer {
       set {
         if (open && value != gpuEnabled) {
           if (value) {
-            Add(new ATI.ATIGroup(settings));
+            //Add(new ATI.ATIGroup(settings));
             Add(new Nvidia.NvidiaGroup(settings));
           } else {
-            RemoveType<ATI.ATIGroup>();
+            //RemoveType<ATI.ATIGroup>();

```

Comment:
Should be undone.

Reply:
Sorry about this -> Thats clearly wrong -> Changed